pkg/recorder/recorder.go

```
@@ -0,0 +1,114 @@
+	r.ctx, r.ctxCancel = context.WithCancel(context.Background())
+	return r
+}
+
+func (r *Recorder) Start() error {
+	if r.Req.Task == plugnmeet.RecordingTasks_START_RECORDING {
+		r.filePath = path.Join(r.AppCnf.Recorder.CopyToPath.MainPath, r.AppCnf.Recorder.CopyToPath.SubPath, r.Req.GetRoomSid())
+		err := os.MkdirAll(r.filePath, 0755)
+		if err != nil {
+			switch {
+			case errors.Is(err, os.ErrExist):
+				log.Infoln("%s already exists", r.filePath)
```

golangci-lint (1.62.2)

printf: github.com/sirupsen/logrus.Infoln call has possible Printf formatting directive %s

(govet)